<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImage">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="actionBrightness"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Stretching"/>
    <addaction name="actionNegative_Image"/>
    <addaction name="actionBiner_Image"/>
    <addaction name="actionHistogram_Grayscale"/>
    <addaction name="actionHistogram_RGB"/>
    <addaction name="actionHistogram_Equalization"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <widget class="QMenu" name="menuMenu_Rotasi">
     <property name="title">
      <string>Menu Rotasi</string>
     </property>
     <addaction name="actionRotasi_Minus_45"/>
     <addaction name="actionRotasi_45"/>
     <addaction name="actionRotasi_Minus_90"/>
     <addaction name="actionRotasi_90"/>
     <addaction name="actionRotasi_180"/>
    </widget>
    <addaction name="actionTranslasi"/>
    <addaction name="menuMenu_Rotasi"/>
   </widget>
   <widget class="QMenu" name="menuResize">
    <property name="title">
     <string>Resize</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionSkewed_Image"/>
    <addaction name="actionCrop"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Spasial">
    <property name="title">
     <string>Operasi Spasial</string>
    </property>
    <widget class="QMenu" name="menuDeteksi_Tepi">
     <property name="title">
      <string>Deteksi Tepi</string>
     </property>
     <addaction name="actionSobel"/>
     <addaction name="actionCanny"/>
    </widget>
    <widget class="QMenu" name="menuMorfologi">
     <property name="title">
      <string>Morfologi</string>
     </property>
     <addaction name="actionDilasi"/>
     <addaction name="actionErosi"/>
     <addaction name="actionOpening"/>
     <addaction name="actionClosing"/>
    </widget>
    <addaction name="menuDeteksi_Tepi"/>
    <addaction name="menuMorfologi"/>
   </widget>
   <widget class="QMenu" name="menuThresholding">
    <property name="title">
     <string>Thresholding</string>
    </property>
    <addaction name="actionBinary"/>
    <addaction name="actionBinary_Invers"/>
    <addaction name="actionTrunc"/>
    <addaction name="actionTo_Zero"/>
    <addaction name="actionTo_Zero_Invers"/>
   </widget>
   <widget class="QMenu" name="menuAdaptive">
    <property name="title">
     <string>Adaptive</string>
    </property>
    <addaction name="actionThresh_Mean"/>
    <addaction name="actionThresh_Gausian"/>
    <addaction name="actionOtsu"/>
   </widget>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuOperasi_Geometri"/>
   <addaction name="menuResize"/>
   <addaction name="menuOperasi_Spasial"/>
   <addaction name="menuThresholding"/>
   <addaction name="menuAdaptive"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Simple Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionNegative_Image">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionHistogram_Grayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_RGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="actionRotasi_Minus_45">
   <property name="text">
    <string>Rotasi Minus 45</string>
   </property>
  </action>
  <action name="actionRotasi_45">
   <property name="text">
    <string>Rotasi 45</string>
   </property>
  </action>
  <action name="actionRotasi_Minus_90">
   <property name="text">
    <string>Rotasi Minus 90</string>
   </property>
  </action>
  <action name="actionRotasi_90">
   <property name="text">
    <string>Rotasi 90</string>
   </property>
  </action>
  <action name="actionRotasi_180">
   <property name="text">
    <string>Rotasi 180</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionSkewed_Image">
   <property name="text">
    <string>Skewed Image</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionDilasi">
   <property name="text">
    <string>Dilasi</string>
   </property>
  </action>
  <action name="actionErosi">
   <property name="text">
    <string>Erosi</string>
   </property>
  </action>
  <action name="actionOpening">
   <property name="text">
    <string>Opening</string>
   </property>
  </action>
  <action name="actionClosing">
   <property name="text">
    <string>Closing</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="text">
    <string>Binary</string>
   </property>
  </action>
  <action name="actionBinary_Invers">
   <property name="text">
    <string>Binary Invers</string>
   </property>
  </action>
  <action name="actionTrunc">
   <property name="text">
    <string>Trunc</string>
   </property>
  </action>
  <action name="actionTo_Zero">
   <property name="text">
    <string>To Zero</string>
   </property>
  </action>
  <action name="actionTo_Zero_Invers">
   <property name="text">
    <string>To Zero Invers</string>
   </property>
  </action>
  <action name="actionThresh_Mean">
   <property name="text">
    <string>Thresh Mean</string>
   </property>
  </action>
  <action name="actionThresh_Gausian">
   <property name="text">
    <string>Thresh Gausian</string>
   </property>
  </action>
  <action name="actionOtsu">
   <property name="text">
    <string>Otsu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
